app/plugins/Wallet/js/helpers.js

```diff
@@ -5,7 +5,7 @@ const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
 // Ensure precision
-BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ DECIMAL_PLACES: 30 });
```

Comment:
I don't think that's a realistic use case, but I guess it's good to support it, because sometimes people do weird things.